src/Helper/Service/IdentityFactory.php

```diff
@@ -50,14 +50,16 @@ private function discoverAuthenticationService(ContainerInterface $container)
             return $container->get(AuthenticationService::class);
         }
 
-        if ($container->has(\Zend\Authentication\AuthenticationService::class)) {
-            return $container->get(\Zend\Authentication\AuthenticationService::class);
+        // @codingStandardsIgnoreStart - Because of non ::class references for Zend
+        if ($container->has('Zend\Authentication\AuthenticationService')) {
+            return $container->get('Zend\Authentication\AuthenticationService');
         }
 
         return $container->has(AuthenticationServiceInterface::class)
             ? $container->get(AuthenticationServiceInterface::class)
-            : ($container->has(\Zend\Authentication\AuthenticationServiceInterface::class)
-                ? $container->get(\Zend\Authentication\AuthenticationServiceInterface::class)
+            : ($container->has('Zend\Authentication\AuthenticationServiceInterface')
```

Comment:
I stay with it: we should just remove this bridge stuff in the next minor anyway

Reply:
This is all gone in 3.0, so it makes sense to me to leave it in 2.x